python3-requirements.json

```diff
@@ -0,0 +1,291 @@
+{
+    "name": "python3-requirements",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [          
+            {  "name": "lapack",
+            "buildsystem": "cmake",
```

Comment:
`cmake-ninja`?

Reply:
`cmake` is how I usually build. Is there a specific reason to use `cmake-ninja` instead?
Again, will give it a go locally first to make sure I won't waste buildbot's time. Thanks for the input in any case!

Reply:
it's faster. Only some cases fail with `cmake-ninja`. But I don't remember lapack being one of them.